tests/components/litterrobot/test_select.py

```diff
@@ -32,7 +33,7 @@ async def test_wait_time_select(hass: HomeAssistant, mock_account):
     ent_reg = entity_registry.async_get(hass)
     entity_entry = ent_reg.async_get(SELECT_ENTITY_ID)
     assert entity_entry
-    assert entity_entry.entity_category == ENTITY_CATEGORY_CONFIG
+    assert entity_entry.entity_category == EntityCategory.CONFIG
```

Comment:
Please use `is` for `entity.entity_category` comparison

Reply:
Agreed, done.